src/consent.js

```diff
@@ -46,12 +46,6 @@ ConsentHandler.prototype.getUserConsentState = function () {
     return userConsentState;
 };
 
-ConsentHandler.prototype.getEventConsentState = function (eventConsentState) {
```

Comment:
@mmustafa-tse  duplicated this method and put it in `common.js`.  It is no longer used here, and not referenced here at all.  tests continue to pass.